doc/Command-Reference.md

```diff
@@ -4088,6 +4092,170 @@ This command deletes the SNMP Trap server IP address to which SNMP agent is expe
+
+With no optional argument, all the ports muxcable status will be displayed in tabular form, or user can pass --json option to display in json format
+
+- Example:
+  ```
+  admin@sonic:~$ show muxcable status
+  PORT        STATUS    HEALTH
+  ----------  --------  --------
+  Ethernet32  active    HEALTHY
+  Ethernet0   auto      HEALTHY
+
+  admin@sonic:~$ show muxcable status --json
+{
+    "MUX_CABLE": {
+        "Ethernet32": {
+            "STATUS": "active",
+            "HEALTH": "HEALTHY"
+        },
+        "Ethernet0": {
+            "STATUS": "auto",
+            "HEALTH": "HEALTHY"
+        }
+    }
+}
+
+  admin@sonic:~$ show muxcable status Ethernet0
+  PORT       STATUS    HEALTH
+  ---------  --------  --------
+  Ethernet0  auto      HEALTHY
+
+  admin@sonic:~$ show muxcable status Ethernet0 --json
+{
+    "MUX_CABLE": {
+        "Ethernet0": {
+            "STATUS": "auto",
+            "HEALTH": "HEALTHY"
+        }
+    }
+}
+  ```
+
+**show muxcable config**
+
+This command displays all the configurations of either all the ports which are connected to muxcable or any individual port selected by the user. The resultant table or json output will show the current configurations of muxcable on the port(active/standby) and also the ipv4 and ipv6 address of the port as well as peer TOR ip address with the hostname.
+
+- Usage:
+  ```
+  show muxcable config [OPTIONS] [PORT]
+  ```
+With no optional argument, all the ports muxcable configuration will be displayed in tabular form
+While displaying the muxcable configuration, users can configure the following fields
+1) optional - Port name should be a valid port
+3) optional - --json option to display the result in json format. By default output will be in tabular format.
+
+- Example:
+  ```
+  admin@sonic:~$ show muxcable config
+  SWITCH_NAME    PEER_TOR
+  -------------  ----------
+  sonic          10.1.1.1
+  port       state    ipv4      ipv6
+  ---------  -------  --------  --------
+  Ethernet0  active  10.1.1.1  fc00::75
+
+  admin@sonic:~$ show muxcable config --json
+{
+    "MUX_CABLE": {
+        "PEER_TOR": "10.1.1.1",
+        "PORTS": {
+            "Ethernet0": {
+                "STATE": "active",
+                "SERVER": {
+                    "IPv4": "10.1.1.1",
+                    "IPv6": "fc00::75"
+                }
+            }
+        }
+    }
+}
+
+  admin@sonic:~$ show muxcable config Ethernet0
+  SWITCH_NAME    PEER_TOR
+  -------------  ----------
+  sonic          10.1.1.1
+  port       state    ipv4      ipv6
+  ---------  -------  --------  --------
+  Ethernet0  active  10.1.1.1  fc00::75
+
+  admin@sonic:~$ show muxcable config Ethernet0 --json
+{
+    "MUX_CABLE": {
+        "PORTS": {
+            "Ethernet0": {
+                "STATE": "active",
+                "SERVER": {
+                    "IPv4": "10.1.1.1",
+                    "IPv6": "fc00::75"
+                }
+            }
+        }
+    }
+}
```

Comment:
Indent 2 spaces to align with block to fix formatting.

Reply:
fixed